terraform/environments/xhibit-portal/network-infrastructure.tf

```diff
@@ -249,6 +276,30 @@ resource "aws_security_group_rule" "waf_lb_allow_web_users" {
   ipv6_cidr_blocks  = ["::/0"]
 }
 
+resource "aws_security_group_rule" "egress-to-prtg" {
+  depends_on               = [aws_security_group.prtg_lb]
+  security_group_id        = aws_security_group.prtg_lb.id
+  type                     = "egress"
+  description              = "allow web traffic to get to prtg"
+  from_port                = 443
+  to_port                  = 443
+  protocol                 = "TCP"
+  source_security_group_id = aws_security_group.portal_server.id
+}
+
+resource "aws_security_group_rule" "prtg_lb_allow_web_users" {
+  depends_on        = [aws_security_group.prtg_lb]
+  security_group_id = aws_security_group.prtg_lb.id
+  type              = "ingress"
+  description       = "allow web traffic to get to ingestion server"
```

Comment:
Can you correct these descriptions? This is access to the PRTG/import machine load balancer